### ISSUE TITLE
[Grid] Aria-expanded attribute of filter and column menus is not updated correctly

### ISSUE THREAD

**Issue by georgi-sla** [telerik/kendo-angular#4544]
### Description

Currently, when the user opens either the filter or column menu, the aria-expanded property of the entire cell and the button responsible for opening the two elements is set to true. However, upon closing the menus, the value of the aria-expanded attribute is not updated and remains true.

### Steps To Reproduce

1. Create a Grid component.
2. Enable column and filter menus.
3. Inspect either the cell or the button responsible for opening the menus.
4. Open either of the two menus and observe the DOM.

Please, check out the following example: https://codesandbox.io/p/sandbox/long-mountain-r9mzdl

Note: To see the issue with the aria-expanded of the button opening the menu(s), make sure to disable the navigable property of the Grid.

### Actual Behavior

The Grid component does not correctly update the value of the aria-expanded attribute of the column and filter menus when the user closes them.

### Expected Behavior

The aria-expanded attribute should be set back to false when the column or filter menu is closed.

### Browser

All

### Browser version

latest

### OS type

MacOS
